runtime/caches/redis.ts

```diff
@@ -81,125 +89,78 @@ export const caches: CacheStorage = {
       ): Promise<ReadonlyArray<Request>> => {
         throw new Error("Not Implemented");
       },
-      /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Cache/match) */
+      delete: async (
+        request: RequestInfo | URL,
+        _?: CacheQueryOptions,
+      ): Promise<boolean> => {
+        console.debug("[cache]", "delete");
```

Comment:
these methods are invoked multiples times per minute, does it worth to add logs in each call?

Reply:
we can instead extract only the necessary info to keep it tracked? (and use a metric instead of a log?)